content/id/docs/concepts/cluster-administration/flow-control.md

```diff
@@ -0,0 +1,374 @@
+---
+title: Prioritas dan Kesetaraan API (API Priority and Fairness)
+content_template: templates/concept
+min-kubernetes-server-version: v1.18
+---
+
+{{% capture overview %}}
+
+{{< feature-state state="alpha"  for_k8s_version="v1.18" >}}
+
+Mengontrol perilaku server API dari Kubernetes pada situasi beban berlebih
+merupakan tugas utama dari administrator klaster. {{< glossary_tooltip term_id="kube-apiserver" text="kube-apiserver" >}} memiliki beberapa kontrol yang tersedia
+(seperti opsi `--max-request-inflight` dan `--max-mutating-request-inflight`
+pada baris perintah atau _command-line_) untuk membatasi jumlah pekerjaan luar biasa yang akan
+diterima, untuk mencegah banjirnya permintaan masuk dari beban berlebih
+yang berpotensi untuk menghancurkan server API. Namun opsi ini tidak cukup untuk memastikan
+bahwa permintaan yang paling penting dapat diteruskan pada saat kondisi lalu lintas yang cukup tinggi.
+
+Fitur Prioritas dan Kesetaraan API atau _API Priority and Fairness Feature_ (APF) adalah alternatif untuk meningkatkan
+batasan _max-inflight_ seperti yang disebutkan di atas. APF mengklasifikasi
+dan mengisolasi permintaan dengan cara yang lebih halus. Fitur ini juga memperkenalkan
+jumlah antrian yang terbatas, sehingga tidak ada permintaan yang ditolak
+pada saat terjadi lonjakan permintaan dalam waktu yang sangat singkat. Permintaan dibebaskan dari antrian dengan menggunakan
+teknik antrian yang adil (_fair queuing_) sehingga, sebagai contoh, perilaku buruk dari satu
+{{<glossary_tooltip text="controller" term_id="controller">}} tidak seharusnya
+mengakibatkan _controller_ yang lain menderita (meskipun pada tingkat prioritas yang sama).
+
+{{< caution >}}
+Permintaan yang diklasifikasikan sebagai "long running" - _primarily watches_ - tidak
```

Comment:
Is it feasible to localize “primarily”? I think a word similar to kebanyakan would make more sense than _primarily_.

Reply:
"terutama" is better.